src/java.base/share/classes/java/lang/ref/ReferenceQueue.java

```diff
@@ -34,6 +34,14 @@
 /**
  * Reference queues, to which registered reference objects are appended by the
  * garbage collector after the appropriate reachability changes are detected.
+ *
+ * <p><a href="{@docRoot}/java.base/java/lang/ref/package-summary.html#MemoryConsistency">Memory consistency effects</a>:
+ * The enqueueing of a reference on a queue (by the garbage collector, or by a
```

Comment:
In passing, I wonder if you're actually meant to say "on a queue" here.  Elsewhere the API docs speak of adding and removing, e.g. the enqueue method speaks of adding a reference "to" the queue.